CorsixTH/Lua/languages/brazilian_portuguese.lua

```diff
@@ -475,12 +476,12 @@ new_game_window = {
 
 tooltip.new_game_window = {
   player_name = "Digite no nome que você deseja ser chamado no jogo",
-  difficulty = "Selecione o nível de dificuldade do jogo.",
+  difficulty = "Selecione o nível de dificuldade do jogo",
   easy = "Se você é novo em jogos de simulaçäo esta é a opçäo para você",
   medium = "Este é o meio do campinho se você está inseguro do que deve escolher",
   hard = "Se você está acostumado com este tipo de jogo e quer mais desafios, escolha esta opçäo",
   tutorial = "Se você precisa de ajuda para começar a jogar, escolha esta opçäo",
-  start = "Iniciar o jogo com a configuraçäo selecionada.",
+  start = "Iniciar o jogo com a configuraçäo selecionada",
   cancel = "Ah, eu näo queria começar um novo jogo!",
 }
```

Comment:
checking actual English text in google this should be Começar for start and dificuldade for difficulty.
I don't understand why one word in English file becomes more detailed and maybe won't fit in the space in the translation